packages/lakka/retroarch_base/retroarch_joypad_autoconfig/joypad_configs/udev/Odin Gamepad.cfg

```diff
@@ -0,0 +1,34 @@
+input_driver = "udev"
+input_device = "Odin Gamepad"
+input_device_display_name = "Odin Gamepad"
+input_vendor_id = "8224"
+input_product_id = "273"
+
```

Comment:
We should add label and add it to [retroarch-joypad-autoconfig repository](https://github.com/libretro/retroarch-joypad-autoconfig) ?
[
Source Android file](https://github.com/libretro/retroarch-joypad-autoconfig/pull/847/files)

Reply:
Dont disagree, but not sure I am ready to commit it yet, as the buttons on the back of the console arent included yet.